presto-main/src/main/java/com/facebook/presto/execution/TaskExecutor.java

```diff
@@ -855,6 +897,16 @@ public long getRunningTasksLevel4()
     }
 
     @Managed
+    public long getLongestRunningSplitMillis()
```

Comment:
@martint pointed out this is ambiguous as to current or historical and suggests `getMaxActiveSplitTime`